examples/todo-list/src/AddTodoTask.js

```diff
@@ -0,0 +1,35 @@
+import { useState } from "react";
+import { useStore } from "./store";
+import { FcTodoList } from "react-icons/fc";
+
+export default function AddTodoTask() {
+  const [counter, setCounter] = useState(0);
+  const [error, setError] = useStore.error();
+  const [todo, setTodo] = useStore.todo();
+  const addTask = (e) => {
+    console.log(JSON.stringify(e.target.children[0].value));
+    e.preventDefault();
+    if (e.target.children[0].value.trim() === "") {
+      setError("Can't add empty tasks");
+      return;
+    } else setError(undefined);
+
+    setTodo({
+      ...todo,
+      [counter]: { text: e.target.children[0].value, id: counter }
```

Comment:
Maybe using an epoch for id's is enough

```suggestion
      [counter]: { text: e.target.children[0].value, id: Date.now() }
```